sw/device/tests/usbdev_stream_test.c

```diff
@@ -35,7 +35,7 @@
 #define NUM_STREAMS USBUTILS_STREAMS_MAX
 #endif
 
-#define TRANSFER_BYTES_SILICON (0x10U << 20)
+#define TRANSFER_BYTES_SILICON (0x10U << 16)
```

Comment:
Please don't reduce this value so far; it will barely be testing anything, and there's a danger that if something makes the streaming massively slower/less efficient, it will go completely unnoticed.

usbdev_stream_test is much faster than usbdev_mixed_test and usbdev_iso_test because it is using Bulk Transfers and running as fast as the bus can go. Just 1MiB completes in 4.8 seconds on my laptop, which is well below the 30s that you proposed originally.

Even the 16MiB originally that is specified for usbdev_stream_test should complete in 38s (and thus not time out with the 60s limit that is presently set).